modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/MessageBoardMessageResourceImpl.java

```diff
@@ -354,6 +355,29 @@ public Page<MessageBoardMessage> getSiteMessageBoardMessagesPage(
 			sorts);
 	}
 
+	public Page<MessageBoardMessage>
+			getSiteUserMessageBoardMessagesActivityPage(
+				Long siteId, Long userId, Pagination pagination)
+		throws Exception {
+
+		int start = QueryUtil.ALL_POS;
+		int end = QueryUtil.ALL_POS;
+
+		if (pagination != null) {
+			start = pagination.getStartPosition();
+			end = pagination.getEndPosition();
+		}
+
+		return Page.of(
+			transform(
+				_mbMessageService.getSiteUserMessageBoardMessagesActivity(
+					siteId, userId, start, end),
+				this::_toMessageBoardMessage),
+			pagination,
+			_mbMessageLocalService.getSiteUserMessageBoardMessagesActivityCount(
```

Comment:
Use _mbMessageService.getSiteUserMessageBoardMessagesActivityCount instead.

Reply:
Remember how you told me maybe it's not necessary? I looked this up and methods that use count in LocalService don't create that method in Service. I talked to you about it @joseabelenda